### ISSUE TITLE
Synthesizers Sound Like Piano

### ISSUE THREAD

**Issue by SquiggDog5** [musescore/MuseScore#22827]
### Issue type

General playback bug

### Bug description

This is just as it sounds--Some, but not all, of the synthesizer instruments play the voice of the piano rather than the synthesizer sound that it's supposed to be after I updated to version 4.3.0. The synthesizers that this occurs in are the poly, saw, square, and string synthesizers. However, the rain synthesizer remained completely intact. But yeah, to sum up, as soon as I updated to 4.3.0, the synthesizers could only play the piano voice. I've tried reinstallation, opening a new file and using the same synthesizers, and resetting the interface to default, but nothing has worked.

### Steps to reproduce

1. In version prior to 4.3.0, have a synthesizer off the mentioned list in the description.
2. Update MuseScore to 4.3.0.
3. The synthesizer now plays piano sounds and can't be changed back to a synthesizer sound.

### MuseScore Version

4.3.0

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11

**Comment by mathesoncalum** [comment]
Duplicate of #22757 (will be fixed in 4.3.1 after #22793 is ported)
